nvtabular/ops.py

```diff
@@ -1226,5 +1226,19 @@ def get_embedding_size(encoders, cat_names):
     return ret_list
 
 
+def get_embeddings(workflow):
+    cols = get_embedding_order(workflow.columns_ctx["categorical"]["base"])
+    return get_embeddings_dask(workflow.stats["categories"], cols) 
+
+
+def get_embeddings_dask(paths, cat_names):
+    embeddings = {}
+    for col in sorted(cat_names):
+        path = encoders[col]
+        num_rows, _, _ = cudf.io.read_parquet_metadata(path)
+        embeddings[col] = (num_rows, _emb_sz_rule(num_rows))
+    return embeddings
```

Comment:
I don't think you need to sort in get_embeddings_dask - we're already sorting in the get_embedding_order call from get_embeddings.

Also since we're not exposing get_embeddings_dask as part of our public API, lets either merge these functions (since they are short) or prefix with a _ . Also I don't think the 'encoders' value on line 1237 is defined ?

```suggestion
def get_embeddings(workflow):
    cat_names = get_embedding_order(workflow.columns_ctx["categorical"]["base"])
    encoders = workflow.stats["categories"]
    embeddings = {}
    for col in cat_names:
        path = encoders[col]
        num_rows, _, _ = cudf.io.read_parquet_metadata(path)
        embeddings[col] = (num_rows, _emb_sz_rule(num_rows))
    return embeddings
```

Reply:
I like the idea of replacement here, but I'd like to suggest either leaving the old name as it is (`get_embedding_size`) or just making it plural (`get_embedding_sizes`).

The proposed name (`get_embeddings`) is confusing since we are using this function to get embedding sizes, not to get embeddings themselves.